fury-on-ovhcloud/README.md

```diff
@@ -0,0 +1,539 @@
+# Fury on OVHcloud
+
+This step-by-step tutorial guides you to deploy the **Kubernetes Fury Distribution** on a Managed Kubernetes cluster on OVHcloud.
+
+This tutorial covers the following steps:
+
+1. Deploy a Managed Kubernetes cluster on OVHcloud with `Terraform`
+2. Download the latest version of Fury with `furyctl`
+3. Install the Fury distribution
+4. Explore some features of the distribution
+5. Teardown of the environment
+
+> ⚠️ OVHcloud **charges you** to provision the resources used in this tutorial. You should be charged only a few euros, but we are not responsible for any costs that incur.
+>
+> ❗️ **Remember to remove all the instances by following all the steps listed in the teardown phase.**
+>
+> 💻 If you prefer trying Fury in a local environment, check out the [Fury on Minikube][fury-on-minikube] tutorial.
+
+## Prerequisites
+
+This tutorial assumes some basic familiarity with Kubernetes and [OVHcloud](https://www.ovhcloud.com/). Some experience with Terraform is helpful but not required.
+
+To follow this tutorial, you need:
+
+- **OVHcloud Account** - You must have an active account to use OVHcloud services. Use this [guide](https://docs.ovh.com/gb/en/customer/create-ovhcloud-account/) to create one.
+- **OVHcloud Public Cloud project** - You must have a Public Cloud Project. Use this [guide](https://docs.ovh.com/gb/en/public-cloud/create_a_public_cloud_project/) to create one.
+- **OVHcloud Openstack User** - To manage the network environment with the Terraform provider, you must have an Openstack user. Use this [guide](https://docs.ovh.com/gb/en/public-cloud/creation-and-deletion-of-openstack-user/) to create one.
+
+## Step 1 - Automatic provisioning of a Managed Kubernetes Cluster in a private network
+
+We are using the `terraform` cli to automaticly deploy the private network, and then the Managed Kubernetes Cluster.
+
+| Terraform Provider | Credentials |
+|---|---|
+| [OVH Provider](https://registry.terraform.io/providers/ovh/ovh/latest/docs) | [OVHcloud API](https://docs.ovh.com/gb/en/api/first-steps-with-ovh-api/) credentials |
+| [Openstack Provider](https://registry.terraform.io/providers/terraform-provider-openstack/openstack/latest/docs) | [Openstack user](https://docs.ovh.com/sg/en/public-cloud/creation-and-deletion-of-openstack-user/) credentials |
+
+### Pre-requisites
+
+The tools we need are `furyctl`, `terraform`, `kubectl` and `kustomize`.
+
```

Comment:
We should mention that you can also use the container that we make, like in the other guides.

And that there's a script provided to run all the installation commands at once.